DEVELOPERS.md

```diff
@@ -29,4 +29,19 @@ that were made, and allow feedback if necessary
 Deployment process:
 When PRs are merged, add the [merged] label to the ticket that it solves. This allows us to make a list of 
 changes for release notes or other documentation. Once the release has been made, remove these labels from 
-these tickets.
\ No newline at end of file
+these tickets.
+
+## Specific notes
+
+### Custom Django permissions
+
+If there is a need for defining custom permissions we should define them in the corresponding model's `Meta` class
+as [described in the Django docs](https://docs.djangoproject.com/en/dev/topics/auth/customizing/#custom-permissions). There
+is no generic model to **host** all custom permissions, but we'll add them to the most closely related model. These
+permissions should then be added to a `Group` so that we can manage them properly in the Django admin. The `Group` should
+be created in the corresponding database migration in which the permission is added to the model.
```

Comment:
now this should be that we create a fixture, right?

Reply:
100%, addressed in c425e660b5edd489c74c9f7b8a6c41ee0056de67